examples/adaptation.ipynb

```diff
@@ -0,0 +1,604 @@
+      "                                    lane data will be interpolated from the existing data\n",
+      "2024-12-09 02:31:42 PM - [damage_network_base.py:127] - root - WARNING - Interpolated the missing lane data as follows: {'secondary': 2.0, 'tertiary': 2.0, 'trunk': 2.0}\n",
+      "c:\\Users\\hauth\\miniforge3\\envs\\ra2ce_env\\Lib\\site-packages\\geopandas\\geodataframe.py:1528: SettingWithCopyWarning: \n",
+      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
+      "Try using .loc[row_indexer,col_indexer] = value instead\n",
+      "\n",
+      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
+      "  super().__setitem__(key, value)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "     AO0_impact  AO1_impact  AO1_benefit  AO2_impact  AO2_benefit  \\\n",
+      "0      0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "1      0.065512    0.065512          0.0    0.006782     -0.05873   \n",
+      "2      0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "3      0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "4      0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "..          ...         ...          ...         ...          ...   \n",
+      "225    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "226    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "227    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "228    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "229    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "\n",
+      "         AO1_cost  AO1_bc_ratio      AO2_cost  AO2_bc_ratio  \n",
+      "0    1.893107e+05           0.0  3.663338e+05  0.000000e+00  \n",
+      "1    1.430648e+06           0.0  2.768437e+06 -2.121409e-08  \n",
+      "2    1.920151e+05           0.0  3.715672e+05  0.000000e+00  \n",
+      "3    9.735977e+04           0.0  1.884003e+05  0.000000e+00  \n",
+      "4    6.490651e+04           0.0  1.256002e+05  0.000000e+00  \n",
+      "..            ...           ...           ...           ...  \n",
+      "225  2.001284e+05           0.0  3.872672e+05  0.000000e+00  \n",
+      "226  2.055373e+05           0.0  3.977339e+05  0.000000e+00  \n",
+      "227  3.515769e+04           0.0  6.803343e+04  0.000000e+00  \n",
+      "228  2.055373e+05           0.0  3.977339e+05  0.000000e+00  \n",
+      "229  3.515769e+04           0.0  6.803343e+04  0.000000e+00  \n",
+      "\n",
+      "[230 rows x 9 columns]\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "_network_config = NetworkConfigWrapper.from_data(None, _network_config_data)\n",
```

Comment:
Explain what is done in this cell.

Reply:
Perhaps move the creation of the network wrapper to the cell where also the config is defined.

Reply:
This will be removed once migrating to the handler